modules/light-clients/08-wasm/keeper/keeper_vm.go

```diff
@@ -45,24 +50,30 @@ func NewKeeperWithVM(
 		panic(errors.New("authority must be non-empty"))
 	}
 
+	sb := collections.NewSchemaBuilder(storeService)
+
 	keeper := &Keeper{
 		cdc:          cdc,
+		vm:           vm,
+		checksums:    collections.NewKeySet(sb, types.ChecksumsKey, "checksums", collections.BytesKey),
 		storeService: storeService,
 		clientKeeper: clientKeeper,
 		authority:    authority,
 	}
 
+	_, err := sb.Build()
```

Comment:
I wonder if we'd be better placed just keeping the schema around on the `Keeper`? 
Avoid having to add it sometime when/if it becomes necessary. 

I don't feel too strongly about it tho, happy to defer to your judgement on this.